great_expectations/core/usage_statistics/anonymizers/checkpoint_run_anonymizer.py

```diff
@@ -59,8 +59,7 @@ def anonymize_checkpoint_run(self, *args, **kwargs) -> Dict[str, List[str]]:
         name: Optional[str] = kwargs.get("name")
         anonymized_name: Optional[str] = self.anonymize(name)
 
-        config_version: Optional[Number] = kwargs.get("config_version")
-        config_version: Optional[str]
+        config_version: Optional[Number, str] = kwargs.get("config_version")
```

Comment:
`Optional[Union[Number, str]]`?

Reply:
@cdkini Yes, in the anonymizer it gets converted to a string...